std/container/rbtree.d

```diff
@@ -1063,6 +1063,8 @@ final class RedBlackTree(T, alias less = "a < b", bool allowDuplicates = false)
      * Insert a single element in the container.  Note that this does not
      * invalidate any ranges currently iterating the container.
      *
+     * Returns: The number of items inserted.
```

Comment:
and here it is called `items`. It needs to be consistent, and Phobos tends to use `elements` to refer to things in ranges and containers.